crowbar.yml

```diff
@@ -73,6 +73,9 @@ nav:
     logs:
       order: 10
       route: 'utils_path'
+    backup:
+      order: 20
+      route: 'backup_path'
```

Comment:
Until we get the wizard I would prefer to keep this link and add a message at the top that clearly states the use case of this page.